spec/lib/rbac/filterer_spec.rb

```diff
@@ -2479,7 +2471,7 @@ def get_rbac_results_for_and_expect_objects(klass, expected_objects)
 
         it "adds .references to the scope" do
           allow(subject).to receive(:warn)
-          expect(resulting_scope.references_values).to eq(["{:miq_server=>{}}", "{:host=>{}}"])
+          expect(resulting_scope.references_values).to eq(%w[miq_servers hosts])
```

Comment:
I think this rspec change best describes the problem with our current implementation:

```diff
-         expect(resulting_scope.references_values).to eq(["{:miq_server=>{}}", "{:host=>{}}"])
+         expect(resulting_scope.references_values).to eq(%w[miq_servers hosts])
```

Before it thought there were 2 tables

Reply:
was it working before?  Is this where it would reference all?

Reply:
it would reference "all"

just looking at it it looks wrong. right?